src/main/webapp/app/exercises/shared/team/team.service.ts

```diff
@@ -178,6 +193,18 @@ export class TeamService implements ITeamService {
         return this.http.get<TeamSearchUser[]>(url, { observe: 'response' });
     }
 
+    /**
+     * Import the teams of an existing source exercise
+     * @param {Exercise} exercise - Exercise the teams should be imported into
+     * @param {Team[]} teams - Exercise the teams should be imported from
+     */
+    importTeamsFromFile(exercise: Exercise, teams: Team[], importStrategyType: TeamImportStrategyType) {
```

Comment:
Would it be better to name the function to something like `importTeams()`? Because here there's a mention of file but the function doesn't do anything with files.